stl/inc/algorithm

```diff
@@ -1364,6 +1364,54 @@ namespace ranges {
 
     inline constexpr _None_of_fn none_of{_Not_quite_object::_Construct_tag{}};
 
+#if _HAS_CXX23
+    class _Contains_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        template <input_iterator _It, sentinel_for<_It> _Se, class _Ty, class _Pj = identity>
+            requires indirect_binary_predicate < ranges::equal_to, projected<_It, _Pj>,
+        const _Ty* > //
+            _NODISCARD constexpr bool operator()(_It _First, _Se _Last, const _Ty& _Val, _Pj _Proj = {}) const {

```

Comment:
There is some whitespace damage here, as the `_NODISCARD` has another level of indentation. I feat we need to disable clang-format here.

Note that the manual line breaks can go in that case